specification/networkanalytics/NetworkAnalytics.Management/examples/2023-11-15/DataProducts_ListRolesAssignments_MaximumSet_Gen.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "DataProducts_ListRolesAssignments_MaximumSet_Gen",
+  "operationId": "DataProducts_ListRolesAssignments",
+  "parameters": {
+    "api-version": "2023-11-15",
+    "subscriptionId": "00000000-0000-0000-0000-00000000000",
+    "resourceGroupName": "aoiresourceGroupName",
+    "dataProductName": "dataproduct01",
+    "body": {}
```

Comment:
>body [](http://example.com/codeflow?start=5&length=4)

why is there an empty body in the example?

Reply:
We are query a managed resource hence we don't require any input from user here.